src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,505 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Main</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600&display=swap"
+        rel="stylesheet">
+  <link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" rel="stylesheet">
+  <script crossorigin="anonymous"
+          integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ"
+          src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body class="container-fluid bg-light">
+<header class="mb-4 row">
+  <div class=" d-flex flex-row align-items-center justify-content-start logo">
+    <img alt="Logo" class="m-4" src="images/logo.svg">
+    <p class="text-primary fs-1">Online Store</p>
+  </div>
+  <nav>
+    <a class=" mx-2 text-decoration-none" href="">
+      <img class="text-primary" src="images/icons/house.svg">
+    </a>
+    <img src="images/icons/chevron-double-right.svg">
+    <a class="text-decoration-none" href="">eCommerce</a>
+    <img src="images/icons/chevron-double-right.svg">
+    <a class="mx-2 text-decoration-none" href="">Electronics</a>
+  </nav>
+</header>
+
+<div class="row">
+  <div class="col-3 col-md-4 filters d-flex flex-column gap-3">
+    <div class=" d-flex flex-row justify-content-between caption">
+      <p class="px-2 fs-5 fw-light">Filters</p>
```

Comment:
major: consider to hide filters on small screens, currently it's list that is hidden